Kitodo/src/main/java/de/sub/goobi/beans/Benutzer.java

```diff
@@ -176,12 +175,18 @@ public void setPasswort(String inpasswort) {
 		this.passwort = inpasswort;
 	}
 
+	/**
+	 * @return add description
```

Comment:
Some IDEs can show a list of open tasks based on keywords. Eclipse for example would list all comments containing `TODO`. Maybe that keyword could be used here, too (either instead of `add description` or additionally).